### PR TITLE
build release in ci

### PR DESCRIPTION
**Pull request by individual-it**
on tag:
1. build an artifact
    moodle wants to have:
    - a ZIP file, GZ or TAR.GZ is not good enough, so we have to install the `zip` tool
    - only one folder in the ZIP file called `ocis`
3. publish it on GitHub
4. publish in the moodle plugin repository

here a successful run: https://drone.owncloud.com/owncloud/moodle-repository_ocis/179/2/4

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -49,3 +51,76 @@ def tests(ctx, tests):
...
+                        "prerelease": len(ctx.build.ref.split("-")) > 1,
+                    },
+                },
+                {
+                    "name": "release to moodle",
+                    "image": OC_UBUNTU,
+                    "environment": {
+                        "ZIPURL": "https://github.com/owncloud/moodle-repository_ocis/releases/download/%s/moodle-repository_ocis_%s.zip" % (ctx.build.ref.replace("refs/tags/", ""), ctx.build.ref.replace("refs/tags/v", "")),
+                        "TAGNAME": ctx.build.ref.replace("refs/tags/v", ""),
+                        "MOODLETOKEN": "eaeab61d20f91fcff3c31da7cb37f864",
```
Comment (individual-it) [inline .drone.star:96]: should come from secret

File: .drone.star [hunk 2]
```diff
@@ -49,3 +51,78 @@ def tests(ctx, tests):
...
+                        "RESPONSE=$(curl https://moodle.org/webservice/rest/server.php \
+                            --data-urlencode \"wstoken=$MOODLETOKEN\" \
+                            --data-urlencode \"wsfunction=local_plugins_add_version\" \
+                            --data-urlencode \"moodlewsrestformat=json\" \
+                            --data-urlencode \"frankenstyle=repository_ocis\" \
+                            --data-urlencode \"zipurl=$ZIPURL\" \
+                            --data-urlencode \"vcssystem=git\" \
+                            --data-urlencode \"vcsrepositoryurl=https://github.com/owncloud/moodle-repository_ocis/\" \
+                            --data-urlencode \"vcstag=$TAGNAME\" \
+                            --data-urlencode \"changelogurl=https://github.com/owncloud/moodle-repository_ocis/releases/tag/$TAGNAME\" \
```
Comment (phil-davis) [inline .drone.star:110]:
```suggestion
                            --data-urlencode \"changelogurl=https://github.com/owncloud/moodle-repository_ocis/releases/tag/v$TAGNAME\" \
```

IMO the GitHub URL needs to have the full release version with the "v" included.

File: .drone.star [hunk 3]
```diff
@@ -49,3 +51,78 @@ def tests(ctx, tests):
...
+                    "commands": [
+                        "RESPONSE=$(curl https://moodle.org/webservice/rest/server.php \
+                            --data-urlencode \"wstoken=$MOODLETOKEN\" \
+                            --data-urlencode \"wsfunction=local_plugins_add_version\" \
+                            --data-urlencode \"moodlewsrestformat=json\" \
+                            --data-urlencode \"frankenstyle=repository_ocis\" \
+                            --data-urlencode \"zipurl=$ZIPURL\" \
+                            --data-urlencode \"vcssystem=git\" \
+                            --data-urlencode \"vcsrepositoryurl=https://github.com/owncloud/moodle-repository_ocis/\" \
+                            --data-urlencode \"vcstag=$TAGNAME\" \
```
Comment (phil-davis) [inline .drone.star:109]: Should this have the git tag name including the "v"?

**Review by individual-it — COMMENTED** [review]
@phil-davis I've change `TAGNAME` to contain `v`

**Review by phil-davis — APPROVED** [review]

**Pull request merged**